### PR TITLE
[flutter_markdown] remove print statements when not handling alignment

### PR DESCRIPTION
**Pull request by alestiago**
Aims to resolve https://github.com/flutter/flutter/issues/110209.

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [X] I signed the [CLA].
- [X] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [ ] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/main/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Comment by alestiago** [comment]
(cc: @goderbauer)

**Comment by alestiago** [comment]
What versioning should this PR receive?

This PR seems about to be merged https://github.com/flutter/packages/pull/2413 and refers to 0.6.11.

File: packages/flutter_markdown/lib/src/builder.dart [hunk 1]
```diff
@@ -757,10 +757,8 @@ class MarkdownBuilder implements md.NodeVisitor {
       case 'pre':
         return styleSheet.codeblockAlign;
       case 'hr':
-        print('Markdown did not handle hr for alignment');
```
Comment (domesticmouse) [inline packages/flutter_markdown/lib/src/builder.dart:760]: Can we use `debugPrint` instead of just deleting?

**Comment by domesticmouse** [comment]
Please resolve conflicts. We can tighten up version once the other issues are resolved and we are ready to land.

Thanks!

File: packages/flutter_markdown/lib/src/builder.dart (hunk 1 above)
Comment (alestiago) [reply packages/flutter_markdown/lib/src/builder.dart:760]: Sure thing. I thought of so, and I agree with the decision. I believe @goderbauer specified deleting but the rationale behind it was to solve analysis warnings. 

I'll change these to debug prints.

**Comment by domesticmouse** [comment]
@alestiago please request a test exemption from Hixie on Discord for this PR. I believe this PR doesn't need additional tests.

**Comment by Hixie** [comment]
Per https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#only-log-actionable-messages-to-the-console I think deleting would be more appropriate?

test-exempt: either removing code or refactor with no semantic change to fix lints.

**Comment by alestiago** [comment]
@domesticmouse shall we revert back to https://github.com/flutter/packages/commit/a5bed325f57b3231281a5ca95de925a43363ee68 ?

**Comment by domesticmouse** [comment]
> @domesticmouse shall I revert back to https://github.com/flutter/packages/commit/a5bed325f57b3231281a5ca95de925a43363ee68 ?

Sgtm

**Comment by alestiago** [comment]
@domesticmouse I've reverted the change and solved the merge conflicts.

**Review by domesticmouse — APPROVED** [review]

**Comment by domesticmouse** [comment]
@stuartmorgan are you happy for this to land without generating a new release?

**Comment by stuartmorgan** [comment]
> @stuartmorgan are you happy for this to land without generating a new release?

Why wouldn't we want package clients to get this fix (which will reduce log spam in production builds)?

**Review by domesticmouse — APPROVED** [review]
PTAL @stuartmorgan

**Comment by domesticmouse** [comment]
Actually, please fix https://github.com/flutter/packages/runs/8266462685

**Comment by domesticmouse** [comment]
@stuartmorgan i believe this is ready to land

**Review by stuartmorgan — APPROVED** [review]
LGTM

**Pull request merged**